opencog/moses/main/populate_atomspace.h

```diff
@@ -56,6 +56,13 @@ void populate(AtomSpace *as, const CTable &ctable);
  * @return ValuePtr containing values such as FloatValue, LinKValues...etc
  */
 ValuePtr vertex_seq_to_value(const vertex_seq& vseq, id::type_node tnode);
+/**
+ *
+ * @param vseq
+ * @param n_rows
+ * @return Opencog::Type
+ */
+Type vertex_seq_to_value(const vertex_seq& vseq, int n_rows);
```

Comment:
@Eman22S this function is declared but not used anywhere. Is it normal?

Reply:
my bad!!